src/WingetCreateCLI/Commands/NewLocaleCommand.cs

```diff
@@ -0,0 +1,326 @@
+                EnsureManifestVersionConsistency(originalManifests);
+                DisplayGeneratedLocales(newLocales);
+
+                if (string.IsNullOrEmpty(this.OutputDir))
+                {
+                    this.OutputDir = Directory.GetCurrentDirectory();
+                }
+
+                string manifestDirectoryPath = SaveManifestDirToLocalPath(originalManifests, this.OutputDir);
+
+                if (ValidateManifest(manifestDirectoryPath))
+                {
+                    if (Prompt.Confirm(Resources.ConfirmGitHubSubmitManifest_Message))
+                    {
+                        return await this.LoadGitHubClient(true) ?
+                            (commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                                originalManifests,
+                                this.GetPRTitle(originalManifests, null, nameof(NewLocaleCommand))))
+                            : false;
+                    }
+                    else
+                    {
+                        Logger.WarnLocalized(nameof(Resources.SkippingPullRequest_Message));
+                    }
+                }
+                else
+                {
+                    return false;
+                }
+
+                return await Task.FromResult(commandEvent.IsSuccessful = true);
+            }
+            catch (Octokit.NotFoundException e)
+            {
+                Logger.ErrorLocalized(nameof(Resources.Error_Prefix), e.Message);
+                Logger.ErrorLocalized(nameof(Resources.OctokitNotFound_Error));
+                return false;
+            }
+            finally
+            {
+                TelemetryManager.Log.WriteEvent(commandEvent);
+            }
+        }
+
+        private static void DisplayGeneratedLocales(List<LocaleManifest> newLocales)
+        {
+            Logger.DebugLocalized(nameof(Resources.GenerateNewLocalePreview_Message));
+            foreach (var localeManifest in newLocales)
+            {
+                Logger.InfoLocalized(nameof(Resources.LocaleManifest_Message), localeManifest.PackageLocale);
+                Console.WriteLine(localeManifest.ToYaml(true));
+            }
+        }
+
+        private static void FillLocalePropertiesForUserCompletion(LocaleManifest fromManifest, LocaleManifest toManifest, List<string> properties)
+        {
+            foreach (string propertyName in properties)
+            {
+                if (propertyName == nameof(LocaleManifest.PackageLocale))
+                {
+                    continue;
+                }
+
+                var value = fromManifest.GetType().GetProperty(propertyName).GetValue(fromManifest);
+                if (value != null)
+                {
+                    toManifest.GetType().GetProperty(propertyName).SetValue(toManifest, value);
+                }
+            }
+        }
+
+        private List<LocaleManifest> ParseArgumentsAndGenerateLocales(Manifests originalManifests)
```

Comment:
I think this method does a lot. I think we should do a couple things to simplify it a bit. 

If a reference locale is provided, we should validate that first then retrieve that locale as our reference locale. If none is provided, then our reference locale should just be our defaultLocaleManifest.

Then this method can be simplified so that it only prompts and generates a single locale manifest:

`private LocaleManifest GenerateLocaleManifest(LocaleManifest referenceLocale)`